kvmagent/kvmagent/test/vm_plugin_testsuite/test_vm_sync.py

```
@@ -36,3 +42,127 @@ def test_vm_sync(self):
+        # mock put_vm_state_to_cache to check its called
+        with mock.patch.object(vm_plugin, 'get_vm_states_from_cache') as get_vm_states_from_cache:
+            rsp = vm_utils.vm_sync()
+            states = rsp.states
+            self.assertEqual(1, len(states))
+            get_vm_states_from_cache.assert_called_once()
+
+        with mock.patch.object(vm_plugin, 'put_vm_state_to_cache') as put_vm_state_to_cache:
+            libvirt_singleton = LibvirtSingleton()
+            conn = libvirt_singleton.conn
+            domain = conn.lookupByName(vm_uuid)
+            vm_utils.VM_PLUGIN.config[kvmagent.SEND_COMMAND_URL] = "http://localhost:7070"
```

Ruff (0.12.2)

`kvmagent` may be undefined, or defined from star imports

(F405)